src/Workspaces/Core/Portable/FindSymbols/Declarations/DeclarationFinder_SourceDeclarations.cs

```diff
@@ -218,8 +261,8 @@ internal static partial class DeclarationFinder
             }
         }
 
-        internal static async Task<ImmutableArray<SymbolAndProjectId>> FindSourceDeclarationsWithPatternInCurrentProcessAsync(
-            Project project, string pattern, SymbolFilter criteria, CancellationToken cancellationToken)
+        private static async Task<ImmutableArray<SymbolAndProjectId>> FindSourceDeclarationsWithPatternInCurrentProcessAsync(
+            string pattern, Func<SearchQuery, Task<ImmutableArray<SymbolAndProjectId>>> searcher)
```

Comment:
should likely be called 'searchAsync' instead of 'searcher'... but i don't care that much :)